index.bs

```diff
@@ -219,6 +348,24 @@ Every [=Document=] has an associated [=ordered set=] of <dfn>previously reported
 </div>
 
 
+Common algorithms {#sec-common-algorithms}
+------------------------------------------
+
+<h4 dfn export>Exposed for paint timing</h4>
+
+<div algorithm="Exposed for paint timing">
+
+    To determine whether an [=/Element=] |element| is [=exposed for paint timing=], given a [=Document=] or null |document|, perform the following steps:
+
+    1. If |element| is not [=connected=], return false.
+    1. If |document| is null, let |document| be [=this=]'s [=relevant settings object=]'s [=relevant global object=]'s [=associated document=].
+    1. If |document| is not [=fully active=], return false.
+    1. If |element|'s [=tree/root=] is not equal to |document|, return false.
```

Comment:
Should be `[=node document=]`

Reply:
My understanding is that this exists to ensure that we don't provide timing for, or references to, elements which have been removed from the DOM sometime between load and paint. In those cases, then its node document will still be |document|, but its root will be the root of some detached tree, and so this should be false.